freyja/sample_deconv.py

```diff
@@ -78,53 +69,35 @@ def map_to_constellation(vals,sample_strains,mapDict):
 				localDict['Other']=vals[jj]
 			else:
 				localDict['Other']+=vals[jj]
-
-	localDict = sorted(localDict.items(), key=lambda x: x[1], reverse=True)#convert to descending order
+	#convert to descending order
+	localDict = sorted(localDict.items(), key=lambda x: x[1], reverse=True)
```

Comment:
This variable implies it's a `dict` but it is actually a `list` of `tuple`. Is that expected?

Reply:
At the moment, I assemble it as a dict (could be changed) and convert to a list of tuples after summarizing by constellation. So, yes… this is expected.